web/ui/react-app/src/pages/targets/ScrapePoolPanel.tsx

```diff
@@ -24,27 +25,14 @@ const ScrapePoolPanel: FC<PanelProps> = ({ scrapePool, targetGroup }) => {
     href: `#${id}`,
     id,
   };
-  const btnProps = {
-    children: `show ${expanded ? 'less' : 'more'}`,
-    color: 'primary',
-    onClick: (): void => setOptions({ expanded: !expanded }),
-    size: 'xs',
-    style: {
-      padding: '0.3em 0.3em 0.25em 0.3em',
-      fontSize: '0.375em',
-      marginLeft: '1em',
-      verticalAlign: 'baseline',
-    },
-  };
 
   return (
     <div className={styles.container}>
-      <h3>
+      <ToggleMoreLess onClick={(): void => setOptions({ expanded: !expanded })}>
```

Comment:
The targets are all expanded by default. But now the button says "show less" on /new/targets for an expanded section, and vice versa. You'll probably want to pass in `expanded` from the outside and keep the state outside of the component, as @boyskila noted in https://github.com/prometheus/prometheus/commit/d8a460f1a684068c4b8961aa33c1f11c82d66d7b#r36603580.